pkg/cmd/server/server.go

```diff
@@ -345,6 +347,34 @@ func (c *Config) Complete(ctx context.Context) (RunnableServer, error) {
 	}, nil
 }
 
+func (c *Config) buildMiddleware(dispatcher dispatch.Dispatcher, ds datastore.Datastore) ([]grpc.UnaryServerInterceptor, []grpc.StreamServerInterceptor) {
+	finalUnaryMiddleware, finalStreamingMiddleware := DefaultMiddleware(log.Logger, c.GRPCAuthFunc, !c.DisableVersionResponse, dispatcher, ds)
```

Comment:
Calling this `final` seems odd since we're appending to it below

Reply:
I've renamed them to `output...`